freqtrade/rpc/telegram.py

```diff
@@ -144,13 +144,16 @@ def send_msg(self, msg: Dict[str, Any]) -> None:
         elif msg['type'] == RPCMessageType.SELL_NOTIFICATION:
             msg['amount'] = round(msg['amount'], 8)
             msg['profit_percent'] = round(msg['profit_percent'] * 100, 2)
+            msg['duration'] = msg['close_date'].replace(
+                microsecond=0) - msg['open_date'].replace(microsecond=0)
 
             message = ("*{exchange}:* Selling {pair}\n"
                        "*Rate:* `{limit:.8f}`\n"
                        "*Amount:* `{amount:.8f}`\n"
                        "*Open Rate:* `{open_rate:.8f}`\n"
                        "*Current Rate:* `{current_rate:.8f}`\n"
                        "*Sell Reason:* `{sell_reason}`\n"
+                       "*Duration:* `{duration}`\n"
```

Comment:
Duration may be included in two forms: as a human-readable 5h 33m 50.3s and in minutes (1350.3 min), separated somehow, like "5h 33m 50.3s / 1350.3 min" or "5h 33m 50.3s (1350.3 min)"